java/src/jmri/jmrix/lenz/XNetReply.java

```diff
@@ -349,6 +369,110 @@ public int getFeedbackEncoderMsgAddr() {
         }
     }
 
+    /**
+     * Returns the number of feedback items in the messages.
+     * For accessory info replies, always returns 1. For broadcast, it returns the
+     * number of feedback pairs. Returns 0 for non-feedback messages.
+     * 
+     * @return number of feedback pair items.
+     */
+    public final int getFeedbackMessageItems() {
+        if (isFeedbackMessage()) {
+            return 1;
+        } else if (isFeedbackBroadcastMessage()) {
+            return (this.getElement(0) & 0x0F);
```

Comment:
That isn’t right.  The second Nibble of the op code is the number of data bytes that follows.  For feedback messages ( directed or broadcast ) each feedback address occupies two bytes. ( an address byte and a data byte ) so the number of feedback values in the message is `(getElement(0)&0x0F)/2`